package.json

```diff
@@ -81,19 +81,19 @@
     "test": "per-env",
     "test:development": "pnpm test:development:web-test-runner",
-    "test:development:playwright": "playwright test --config ./src/playwright/playwright.config.ts --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
-    "test:development:playwright:update": "rimraf ./src/aria-snapshots && playwright test --config ./src/playwright/playwright.config.ts --quiet --project aria --update-snapshots --update-source-method=overwrite --workers 100%",
+    "test:development:playwright": "npm-run-all --parallel test:development:playwright:coverage test:development:playwright:ui",
+    "test:development:playwright:coverage": "http-server ./dist/playwright/coverage-report --port 6008 --silent",
+    "test:development:playwright:ui": "playwright test --config ./src/playwright/playwright.config.ts --ui-port 6007 --update-snapshots --update-source-method=overwrite",
+    "test:development:playwright:update": "playwright test --config ./src/playwright/playwright.config.ts --quiet --project accessibility --update-snapshots --update-source-method=overwrite --workers 100%",
     "test:development:web-test-runner": "web-test-runner --watch",
     "test:production": "pnpm start:production:esbuild && npm-run-all --aggregate-output --print-label --parallel test:production:*",
-    "test:production:playwright:aria": "playwright test --config ./src/playwright/playwright.config.ts --quiet --only-changed origin/main",
-    "test:production:playwright:visuals": "playwright test --config ./src/playwright/playwright.config.ts --ignore-snapshots --quiet --only-changed origin/main",
-    "test:production:playwright:visuals:comment": "Snapshots are ignored because this is meant to be only a quick smoke test telling us when we've written a test incorrectly.",
+    "test:production:playwright": "playwright test --config ./src/playwright/playwright.config.ts --grep 'is accessible|@eslint|@events|@forms|@keyboard|@miscellaneous|@mouse|@stylelint'",
+    "test:production:playwright:comment": "Used locally by developers and as a pre-push hook. Running every '@accessibility' test would take too long. So we only run the 'is accessible' test, which in many cases completes code coverage.",
     "test:production:web-test-runner": "web-test-runner",
     "typecheck": "per-env",
     "typecheck:development": "tsc --erasableSyntaxOnly --outDir ./dist -w",
     "typecheck:production": "tsc --erasableSyntaxOnly --outDir ./dist",
-    "postinstall": "is-ci || pnpm dlx playwright@1.54.2 install --with-deps chromium",
-    "postinstall:comment": "Not installing Chromium in CI every time we run `pnpm install` speeds up builds at the cost of having to also update Playwright's version in our workflows whenever we update Playwright here.",
+    "postinstall": "is-ci || pnpm dlx playwright@1.54.2 install --no-shell --with-deps chromium firefox webkit",
```

Comment:
As Glide Core becomes more widely used, it makes sense to expand testing to other browsers. But I've been putting it off because:

1. Firefox and Safari don't work out of the box with Web Test Runner (though part of the issue may be on our end). 
2. Web Test Runner doesn't support intra-suite parallelization or sharding. So we'd have to run a separate test job for each browser. Otherwise, our CI times would blow up.

Playwright is able run our tests in all three browsers without issue. And it supports parallelization of all kinds. So at least the time-cost of testing all three in CI should be negligible. 

But that's why all three are installed now.  ARIA snapshots are also taken for all three.